tools/deletebyquery/deletebyquery.py

```diff
@@ -160,16 +161,15 @@ def do_solr_query(query):
             break
         else:
             next_cursor_mark = solr_response.result.nextCursorMark
-
-        doc_ids.extend([uuid.UUID(doc['id']) for doc in solr_response.result.response.docs])
+        ids = [uuid.UUID(doc['id']) for doc in solr_response.result.response.docs]
+        delete_from_cassandra(ids)
```

Comment:
you moved the cassandra delete into the `do_solr_query` function? This seems like it would break things. `do_solr_query` is collecting all of the tile ids that need to be deleted. Once that is done it asks the user to confirm the number of documents before executing the delete. With this new flow, the cassandra rows get removed before the solr docs and a user could decide to cancel without realizing data has already been deleted.

Reply:
@frankinspace Whoops, good point. I'll refactor this